Framework/API/test/SampleTest.h

```diff
@@ -261,7 +261,10 @@ class SampleTest : public CxxTest::TestSuite {
                       Mantid::PhysicalConstants::getNeutronAtom(23, 0), 0.072);
     Sample sample;
     IObject_sptr shape = Mantid::Geometry::ShapeFactory().createShape("");
```

Comment:
@martyngigg said to change to `ShapeFactory::createShape` to return CSGObject not IObject types, can you do this so that you can remove the dynamic_cast below.